virtme_ng/utils.py

```diff
@@ -27,3 +40,46 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+def get_conf_obj():
+    """Return virtme-ng main configuration, returning the default if not found."""
+
+    # First check if there is a config file in the user's home config
+    # directory, then check for a single config file in ~/.virtme-ng.conf and
+    # finally check for /etc/virtme-ng.conf. If none of them exist, return the
+    # default configuration.
+    conf_paths = (
+        CONF_FILE,
+        Path(Path.home(), ".virtme-ng.conf"),
+        Path("/etc", "virtme-ng.conf"),
+    )
+    for conf_path in conf_paths:
+        if conf_path.exists():
+            with open(conf_path, encoding="utf-8") as conf_fd:
+                conf = json.loads(conf_fd.read())
+                return conf
+    return CONF_DEFAULT
```

Comment:
mmh, I think that's not enough: some people might already have this file, but it might not contain new entries added in a newer version. In this case, your new helper here will return the content of the file, but `get_conf()` will return `None` + a warning. (see below)

I think we can avoid the warning by either asking `get_conf()` caller to provide the default value, or by looking at the default value in `CONF_DEFAULT`. WDYT?

Reply:
Ah, good catch. I think it would be best to simply return the default. I'll give it a spin.

Reply:
On one hand, `CONF_DEFAULT` regroups all the default values for the config file in the same place. If the caller provides the default value, this value would be closer to where it is used. Both versions have pros and cons :)